File: Đề NĐ các năm/20-21/Bai XTB 2021/I91201/WORK.cpp
#include <iostream>
#include<fstream>
using namespace std;

int main()
{
   int n;
   int a[1000000];
   
   int s=0;
   ifstream f ("WORK.inp");
   f>>n;
   for (int i=0;i<n;i++)
    f>>a[i];
   ofstream g ("WORK.out");
   for (int i=0;i<n;i++)
   {

    if (a[i]>0 )
    {
s =s +a[i];
    }
   }
   g<<s;
   f.close();
   g.close();
    return 0;
}

File: Đề NĐ các năm/20-21/Bai XTB 2021/I36203/WORK.pas
program work;
var T,P:array[1..1000000] of byte;
    i,n,s:integer;
    fi,fo:text;
Begin
  assign(fi,'work.inp'); reset(fi);
  assign(fo,'work.out'); rewrite(fo);
  readln(fi,n);
  for i:=1 to n do read(fi,T[i]);
  for i:=1 to n-1 do read(fi,P[i]);
  for i:=1 to n do
        if i mod 2<>0 then s:=s+P[i];
  s:=S+T[n];
  write(fo,s);
  close(fi); close(fo);
End.